src/main.cpp

```diff
@@ -1239,11 +1276,28 @@ bool CheckTransactionWithoutProofVerification(const CTransaction& tx, CValidatio
             return state.DoS(100, error("CheckTransaction(): txout.nValue too high"),
                              REJECT_INVALID, "bad-txns-vout-toolarge");
         nValueOut += txout.nValue;
-        if (!MoneyRange(nValueOut))
-            return state.DoS(100, error("CheckTransaction(): txout total out of range"),
-                             REJECT_INVALID, "bad-txns-txouttotal-toolarge");
```

Comment:
This is technically a breaking consensus rule change, as it's in theory possible to have sufficient outputs in a transaction that we would overflow the money range and end up with the total being within the money range. Unlikely to actually occur (as the resulting transactions would be too large for a block, though might still be able to reach the mempool), but let's keep this per-output check in place instead of only checking after summing all transparent outputs.